script/DeploySystem.s.sol

```diff
@@ -135,21 +135,26 @@ contract DeploySystem is Deploy {
     }
 
     function deploySystemConfigGlobal() public broadcast returns (address addr_) {
-        string memory filePath = string(abi.encodePacked(
-            "nitro-validator/deployments/",
-            vm.toString(block.chainid),
-            "-nitro-validator-deploy.json"
-        ));
-
-        if (!vm.exists(filePath)) {
-            revert("NitroValidator.json not found. Please deploy nitro-validator first.");
+        console.log("Deploying CertManager / NitroValidator implementations");
+        bytes memory certManager = vm.parseJsonBytes(
+            vm.readFile("nitro-validator/out/CertManager.sol/CertManager.json"), ".bytecode.object");
+        bytes memory nitroValidator = vm.parseJsonBytes(
+            vm.readFile("nitro-validator/out/NitroValidator.sol/NitroValidator.json"), ".bytecode.object");
+
+        bytes32 salt = _implSalt();
+        address certManagerAddress;
+        address nitroValidatorAddress;
+        assembly{
+            certManagerAddress := create2(0, add(certManager, 0x20), mload(certManager), salt)
+            if iszero(extcodesize(certManagerAddress)) { revert(0, 0) }
+//            nitroValidatorAddress := create2(0, add(nitroValidator, 0x20), mload(nitroValidator), salt)
```

Comment:
TODO need to figure out how to pass the `certManagerAddress` as a constructor arg